ShoppingM8-be/src/main/java/edu/pw/shoppingm8/list/api/dto/ListDto.java

```diff
@@ -0,0 +1,15 @@
+package edu.pw.shoppingm8.list.api.dto;
+
+import edu.pw.shoppingm8.list.List;
+import lombok.Value;
+
+@Value
+public class ListDto {
+    Long id;
+    String name;
+    Long ownerId;
```

Comment:
I would prefer to use user dto here - it will be very useful for presentation to display data of user such as name or profile picture, when invitation to list is sent.

Reply:
Chaged